docs/structured-streaming-programming-guide.md

```diff
@@ -2812,6 +2812,19 @@ See [Input Sources](#input-sources) and [Output Sinks](#output-sinks) sections f
 
 # Additional Information
 
+**Notes**
+
+- There're couple of configurations which are not modifiable once you run the query. If you really want to make changes for these configurations, you have to discard checkpoint and start a new query.
```

Comment:
There're -> "There are a". This should probably be reworded though; doesn't need the 2nd person. "Several configurations are not modifiable after the query has run. To change them, discard the checkpoint and start a new query. These configurations include:"

I'm not sure how much we need to explain _why_ they're not modifiable, but suggesting alternatives is good. I think the example is redundant below.

Reply:
Reworded expression looks shorter and better. Will address.

> I'm not sure how much we need to explain why they're not modifiable

Yeah, I agree this is something everyone has different view. Someone could think the page is a guide so don't need to include implementation details. Other one (including me) could think the information is described nowhere so need to be included altogether, especially this is the major difference between batch query and streaming query. 

No strong opinion (verbose vs less wondering on end users) so I can follow up the decision. If you think this is redundant please let me know so that I can address.

> I think the example is redundant below.

Will address.